City_CRUD_API/models.py

```diff
@@ -0,0 +1,15 @@
+from sqlalchemy import Column, Integer, String
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class City(Base):
+    __tablename__ = "city"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(255), nullable=False, unique=True)
+    additional_info = Column(String(511))
+
+    # Backward relationship to Temperature
+    temperatures = relationship("Temperature", back_populates="city")
```

Comment:
Ensure that the `Temperature` model and its relationship with `City` are correctly defined in your codebase to avoid runtime errors.